docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+# Creating Geometry
+
+````@example creating_geometry
+import GeoInterface as GI
+import GeometryOps as GO
+import GeoFormatTypes as GFT
+import CoordinateTransformations
+import Proj
+using CairoMakie
+using GeoMakie
+using GeoJSON
+Makie.set_theme!(Makie.MAKIE_DEFAULT_THEME) # hide
+````
+
+The first thing we need to do is decide which [Coordinate Reference System (CRS)](https://en.wikipedia.org/wiki/Spatial_reference_system) we will be working in. Here, we start with the most common geographic CRS (i.e. coordinates of latitude and longitude), [WGS84](https://epsg.io/4326).
```

Comment:
At this point in the tutorial, I still don't quite understand the point of `CRS`. After introducing it, you show in the first few plots that you don't really need it to get the correct points. I would either add a note that we will return to the importance of CRS or I would introduce it later.

Reply:
That's a good point.  I think the tutorial started as "how to create a geometry with CRS" then evolved to "how to create geometry".  

@alex-s-gardner I guess we could move this a bit further down?

Reply:
Ya, the crs stuff is a bit strange. I think as the ecosystem matures it will make more sense. What if introduce `crs` right before we make our first GeoAxis?

Reply:
That sounds good to me, it's right next to its use there as well.

Reply:
That sounds great to me!